soccer/planning/RRTPlanner.cpp

```diff
@@ -230,10 +238,15 @@ Geometry2d::Point pow(Geometry2d::Point &p1, float i)
 }
 
 using namespace Eigen;
+
+float getTime(Planning::InterpolatedPath &path, int index, const MotionConstraints &motionConstraints, float startSpeed, float endSpeed) {
```

Comment:
Do you think `getDuration()` might be a better name for this?  My first thought when I saw getTime was that it was checking for what point in time something happened like reaching a certain point.